README.md

```diff
@@ -136,17 +136,6 @@ The test suite includes a smoke test for `createdevdata`, so you can easily
 verify that the command is working without disrupting your own development
 environment.
 
-### Live reload
-
-The default gulp `watch` task uses `gulp-livereload` to automatically trigger a
-browser refresh when changes to the frontend code are detected. In order to take
-advantage of this, you will need to install the [LiveReload Chrome
-extension](https://chrome.google.com/webstore/detail/livereload/jnihajbhpnppcggbcgedagnkighmdlei?hl=en).
-
-Once you've installed the extension, simply load the development site in your
-browser (`localhost:8000`) and click the LiveReload extension icon to initiate
-live reloading.
-
```

Comment:
I didn't personally take advantage of this live reloading feature.. seems a shame, though. I wonder if there are ways to get this same thing in webpack? If not for STN then maybe some of our other sites. It does seem handy and very, you know, "modern web dev toolchain"-y.

Reply:
Yeahhhh. There's definitely ways of making this work with Webpack (and with React, there's even ways to _just_ reload the component that gets changed) but I know Emily was fussing with it and never got it to work, so I imagine it's kind of a pain 😬